backend/src/features/hall-of-fame/hall-of-fame.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../../prisma.service';
+import { ScoreEntryDto } from './model/score.entry.dto';
+
+@Injectable()
+export class HallOfFameService {
+  constructor(private prisma: PrismaService) {}
+
+  async getList(): Promise<ScoreEntryDto[]> {
```

Comment:
ist es eigentlich üblich in nestJS mit async und Promise zu arbeiten, statt wie in Angular mit RXJS und Observables? - einfach nur so die frage ;)

Reply:
Ja, in nestJS arbeitest du in erster Linie mit Promise